open-metadata-implementation/access-services/asset-lineage/asset-lineage-server/src/main/java/org/odpi/openmetadata/accessservices/assetlineage/handlers/GlossaryContextHandler.java

```diff
@@ -101,6 +112,17 @@ private void addSchemaElementsContext(String userId, EntityDetail glossaryTerm,
     }
 
 
+    private void addGlossaryCategories(String userId,
+                                       EntityDetail glossaryTerm,
+                                       AssetContext glossaryContext,
+                                       List<Relationship> termCategorization) throws OCFCheckedExceptionBase {
+        if (CollectionUtils.isEmpty(termCategorization)) {
```

Comment:
Would it not be better to have a positive if and put the subsequent line in the body of the if. So you do not need to issue the return?

Reply:
I consider an early return makes code more readable and clean.

Reply:
@danielaotelea in this case I agree with @davidradl 
It is better to change the condition and do nothing if it is not met then do "return;"